spring-boot-modules/spring-boot-springdoc/pom.xml

```diff
@@ -10,9 +10,10 @@
     <description>Project for Springdoc integration</description>
 
     <parent>
-        <groupId>com.baeldung.spring-boot-modules</groupId>
-        <artifactId>spring-boot-modules</artifactId>
-        <version>1.0.0-SNAPSHOT</version>
+        <groupId>com.baeldung</groupId>
```

Comment:
Please update similar with the other modules under spring-boot-modules.

Reply:
Thanks Alvin. Updated